packages/@aws-cdk/aws-appmesh/lib/virtual-gateway-listener.ts

```diff
@@ -58,22 +58,22 @@ export abstract class VirtualGatewayListener {
   /**
    * Returns an HTTP Listener for a VirtualGateway
    */
-  public static httpGatewayListener(props: HttpGatewayListenerProps = {}): VirtualGatewayListener {
-    return new HttpGatewayListener(props);
+  public static http(options: HttpGatewayListenerOptions = {}): VirtualGatewayListener {
+    return new HttpGatewayListener(options);
   }
 
   /**
    * Returns an HTTP2 Listener for a VirtualGateway
    */
-  public static http2GatewayListener(props: HttpGatewayListenerProps = {}): VirtualGatewayListener {
-    return new Http2GatewayListener(props);
+  public static http2(options: HttpGatewayListenerOptions = {}): VirtualGatewayListener {
+    return new Http2GatewayListener(options);
   }
 
   /**
    * Returns a GRPC Listener for a VirtualGateway
    */
-  public static grpcGatewayListener(props: GrpcGatewayListenerProps = {}): VirtualGatewayListener {
-    return new GrpcGatewayListener(props);
+  public static grpc(options: GrpcGatewayListenerOptions = {}): VirtualGatewayListener {
+    return new GrpcGatewayListener(options);
```

Comment:
Also, while we're in this area...

Can you remove all of the `protected` members of this class? I think we only need the `bind` method here, and I want to keep the surface area of it to a minimum.

Reply:
Let's do it